packages/fluentui/react/src/themes/teams/components/Menu/menuItemStyles.ts

```diff
@@ -411,7 +413,17 @@ const menuItemStyles: ComponentSlotStylesPrepared<MenuItemPropsAndState, MenuVar
       top: pxToRem(6),
       right: pxToRem(2),
       left: 'unset'
-    })
+    }),
+
+    backgroundImage: p.vertical ? svgContentVertical(v.color) : svgContentHorizontal(v.color),
```

Comment:
```suggestion
    backgroundImage: svgIndicator(v.color, p.vertical)
```

```suggestion
    backgroundImage: svgIndicator(v.color, p.vertical ? 'up' : 'right')
```

Not fully sure about it

Reply:
I like the first option, we will have them in one function, hence avoiding two files. Would not like the second one, in order to avoid up, right terms which are not rtl safe..